### DISCUSSION TITLE
I can't use mitmproxy to proxy other url

### DISCUSSION THREAD

**Discussion by 19861212** [mitmproxy/mitmproxy#7032] [Q&A]
I haved install mitmproxy on my linux service. I use "mitmweb -vvv --web-host 0.0.0.0 --web-port 8088 --listen-host 0.0.0.0 --listen-port 8000 --set block_global=false" to start my service. When I use "curl -v -x http://myhost:8000 http://www.example.com",I got some message for example "> GET http://www.example.com/ HTTP/1.1

```
> Host: www.example.com
> User-Agent: curl/8.7.1
> Accept: */*
> Proxy-Connection: Keep-Alive
>
* Recv failure: Connection was reset
* Closing connection
curl: (56) Recv failure: Connection was reset"
```

I got only message "client connect
client disconnect" on my mitmproxy service.

**Comment by 19861212** [comment]
![1](https://github.com/user-attachments/assets/c49eb40c-45e1-4b6d-87a2-cb5ae03fbe4e)
![2](https://github.com/user-attachments/assets/38ae10bd-93a1-48c6-aa88-b9982bab7701)
You can see this pictures.

**Reply by Prinzhorn** [reply]
Are you using mitmproxy 10.3.1?

**Reply by 19861212** [reply]
yes.
My mitmproxy version is 10.3.1.
![3](https://github.com/user-attachments/assets/de82f4f8-8407-4493-b63e-21962f0553d9)

**Reply by 19861212** [reply]
I am using a server on Azure.

**Reply by Prinzhorn** [reply]
Add `mitmweb --set proxy_debug=true`, this will log A LOT OF STUFF. It might get you more details than just "client disconnect".

**Reply by 19861212** [reply]
Thank you.Now I can get more messages,but I can't see error message.
![1](https://github.com/user-attachments/assets/620a6408-39dc-4d2f-8513-fe4d4936fb8d)

**Reply by 19861212** [reply]
It's very strange that when I visit Baidu and CSDN, I can get more content.
![2](https://github.com/user-attachments/assets/a857da05-1a91-4bf9-a730-8c01195762ba)

**Reply by 19861212** [reply]
Sorry, it was an error caused by our company's network restrictions.
